public/components/datasources/components/manage/integrations/installed_integrations_table.tsx

```diff
@@ -141,8 +202,20 @@ export const InstalledIntegrationsTable = ({
     <>
       <EuiSpacer />
       <EuiPanel>
-        {integrations.length > 0 ? integrationsTable : <NoInstalledIntegrations />}
+        {integrations.length > 0 ? (
+          integrationsTable
+        ) : (
+          <NoInstalledIntegrations toggleFlyout={toggleFlyout} />
+        )}
       </EuiPanel>
+      {showAvailableFlyout ? (
+        <InstallIntegrationFlyout
```

Comment:
Entrypoint for changes: showing the Flyout from the table

Reply:
BTW, does anyone know why this doesn't violate rule of hooks? I would expect the conditional flyout w/ props in it is a good candidate to cause issues with useState